code/modules/surgery/surgery.dm

```diff
@@ -1,31 +1,46 @@
 /datum/surgery
+	///The name of the surgery operation
 	var/name = "surgery"
-	var/desc = "surgery description"
+	///The description of the surgery, what it does.
+	var/desc
+
+	///Selection: SURGERY_IGNORE_CLOTHES | SURGERY_SELF_OPERABLE | SURGERY_REQUIRE_RESTING | SURGERY_REQUIRE_LIMB | SURGERY_REQUIRES_REAL_LIMB
```

Comment:
Comment what the file that has these is so you can read the documentation there as well